### PR TITLE
Fixes typos, cleans up code

### PR DESCRIPTION
**Pull request by MarinaGryphon**
## About The Pull Request
- Fixes a lot of grammar and punctuation issues.
- Refactors or removes a lot of duplicated code.
- Modularises F13 food/plants into the Fallout module.
- Replaces a lot of uses of `<span class='(warning|notice|danger)'>` with the proper macros.
- Remove a bunch of unused Citadel code.
- Remove a bunch of unused Citadel custom items.

## Why It's Good For The Game
It's been bugging me for years at this point, and the rebase just made it worse.

## Changelog
:cl:
fix: fixed a ton of grammar/punctuation/spelling errors
/:cl:

### CODE REVIEW

File: code/modules/integrated_electronics/subtypes/input.dm [hunk 1]
```diff
@@ -711,8 +711,8 @@
 			LM.playsound_local(get_turf(src), 'sound/machines/triple_beep.ogg', ASSEMBLY_BEEP_VOLUME, TRUE)
 
 /obj/item/integrated_circuit/input/ntnet_packet
-	name = "NTNet networking circuit"
-	desc = "Enables the sending and receiving of messages over NTNet via packet data protocol."
+	name = "wifi networking circuit"
```
Comment (buffyuwu) [inline code/modules/integrated_electronics/subtypes/input.dm:714]: all these mentions of wifi, should it just be 'network' to be vague? or 'radio'? :eyes:

File: code/modules/projectiles/ammunition/caseless/_caseless.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /obj/item/ammo_casing/caseless
-	desc = "A caseless bullet casing."
+	desc = "caseless round"
```
Comment (gurking) [inline code/modules/projectiles/ammunition/caseless/_caseless.dm]: This is the description, not the name.

Comment (MarinaGryphon) [reply code/modules/projectiles/ammunition/caseless/_caseless.dm]: Yeah, oops. I fixed it but then force-pushed over my changes.

Comment (MarinaGryphon) [reply code/modules/projectiles/ammunition/caseless/_caseless.dm]: Looks like this was already fixed, actually. It's now `A caseless round.` Caseless casing makes no sense.

File: code/modules/integrated_electronics/subtypes/input.dm (hunk 1 above)
Comment (MarinaGryphon) [reply code/modules/integrated_electronics/subtypes/input.dm:714]: The issue with 'radio' is that radios don't have device addresses. I could always just change it to 'wireless networking circuit'?

**Pull request merged**